alacritty/src/renderer/mod.rs

```diff
@@ -411,12 +411,12 @@ struct InstanceData {
     g: f32,
     b: f32,
     // Background color.
-    bg_r: f32,
-    bg_g: f32,
-    bg_b: f32,
-    bg_a: f32,
+    bg_r: u8,
+    bg_g: u8,
+    bg_b: u8,
+    bg_a: u8,
```

Comment:
It seems strange that only foreground color broke something, but background color was fine?

Reply:
Yes. I don't claim to understand it, but that's what's going on. For all types that work without a problem, I've picked the smallest size that doesn't negatively impact performance. I've tested various different combination including eliminating **all** bytes, but that didn't improve anything so there's no reason why we'd do that.